File: Tables/XpTranLog.cs
using System;
using System.Collections.Generic;

namespace HrAdm.Tables
{
    public partial class XpTranLog
    {
        public int Sn { get; set; }
        public string RowId { get; set; } = null!;
        public string TableName { get; set; } = null!;
        public string? ColName { get; set; }
        public string? OldValue { get; set; }
        public string? NewValue { get; set; }
        public string Act { get; set; } = null!;
        public DateTime Created { get; set; }
    }
}

File: wwwroot/js/view/XpTranLog.js
var _me = {

    init: function () {        
        //datatable config
        var config = {
            columns: [
                { data: 'TableName' },
                { data: 'ColName' },
                { data: 'RowId' },
                { data: 'Act' },
                { data: 'OldValue' },
                { data: 'NewValue' },
                { data: 'Created' },
            ],
            columnDefs: [
				{ targets: [6], render: function (data, type, full, meta) {
                    return _date.dtsToUiDt(data);
                }},
            ],
        };

        //initial
        new CrudR(config);
    },

}; //class